example-raft-kv/src/store/mod.rs

```diff
@@ -90,8 +109,8 @@ impl RaftStorage<ExampleRequest, ExampleResponse> for ExampleStore {
 
     #[tracing::instrument(level = "trace", skip(self))]
     async fn save_vote(&self, vote: &Vote) -> Result<(), StorageError> {
+        // TODO: What `h` stands for?
```

Comment:
Previously this method is `save_hard_state()`.
`openraft` removed concept `hard_state` and use `vote` instead.

The local variable name is not updated yet. :(

Reply:
Hmm, maybe leave the comment there to remind us that it should be changed in the future. I will resolve this topic.